### ISSUE TITLE
Critical error - array_key_exists() expects parameter 2 to be array, null given 

### ISSUE THREAD

**Issue by micso** [shipperhq/module-shipper#83]
Magento v2.3.2
Module v2.39.5

`[2020-12-10 11:49:53] main.CRITICAL: ShipperHQ save order package error: Warning: array_key_exists() expects parameter 2 to be array, null given in /var/www/html/vendor/shipperhq/module-shipper/src/Helper/Package.php on line 249`

This error each time appears in system.log after the order creation in the admin panel.

**Comment by ibraheemnabeelfauzi** [comment]
Hello @micso can you please tell us what version of PHP is your server running on?

**Comment by micso** [comment]
Hi, I'm using PHP 7.2.34

**Comment by micso** [comment]
Any idea when the module will be fixed? My client is still waiting for a response.

**Comment by wsadasmit** [comment]
Hi misco, I wasn't able to replicate your error simply by following the steps you described. After setting up a fresh
Magento v2.3.2 (php 7.2.34) and installing Module v2.39.5, I placed an order from the admin order section and was able to see no errors reported in var/log/system.log.

I tried a few different ways of placing orders, with and without the dimensional feature enabled, using a new order and reordering previously placed orders. I've scheduled to have this reviewed by the dev team. I'll be sure they know the urgency here, thanks for emphasizing.
